worlds/alttp/ItemPool.py

```diff
@@ -250,8 +250,8 @@ def generate_itempool(world):
         world.push_item(loc, ItemFactory('Triforce Piece', player), False)
         world.treasure_hunt_count[player] = 1
         if world.boss_shuffle[player] != 'none':
```

Comment:
this should check if `isinstance(world.boss_shuffle[player].value, str)`. this fix is incorrect.

Reply:
No idea what's going on then. All you.